dailymotion-player-objc/DMPlayerViewController.m

```diff
@@ -272,6 +282,12 @@ - (void)loadVideo:(NSString *)videoId withParams:(NSDictionary *)params {
 - (void)api:(NSString *)method arg:(NSString *)arg {
     if (!self.inited) return;
     if (!method) return;
+
+    NSString *warnMessage = [self getAPIReadyWarnMessageForMethod:method];
+    if (!self.started && warnMessage != nil) {
```

Comment:
No need for explicit `!= nil`, ObjC can deal with falsey checks, just `if (!self.started && warnMessage) {}`